src/legends/ramp.js

```diff
@@ -21,8 +23,9 @@ export function legendRamp(scale, {
       .attr("width", width)
       .attr("height", height)
       .attr("viewBox", [0, 0, width, height])
-      .style("overflow", "visible")
-      .style("display", "block");
+      .each(addStyle(style));
+    if (svg.style("overflow") === "") svg.style("overflow", "visible");
+    if (svg.style("display") === "") svg.style("display", "block");
```

Comment:
Can we move the overflow: visible; and display: block; to the inline stylesheet so that we don’t need this conditional check?

Reply:
hmm… ramps don't have an internal css yet (and no className either); and there might be a similar problem with css priorities.